libsrc/target/aquarius/far/__far_page.asm

```diff
@@ -15,21 +15,19 @@ __far_page:
     and     a
     jr      z,localfar
     ld      d,e
-    dec     d
```

Comment:
No need to -1 as actual RAM page numbers are now used. Just need to shift the page number.